FSAgenda.php

```diff
@@ -276,25 +276,25 @@ public function updateAttendee(string $url, string $usermail, bool $add, ?string
             }
         }
         
-        $new_etag = $this->caldav_client->updateEvent($url, $etag, $vcal->serialize());
-        if($new_etag === false) {
+        $new_ETag = $this->caldav_client->updateEvent($vCalendarFilename, $ETag, $vCalendar->serialize());
```

Comment:
It's a detail, but for consistency you might also want to apply a similar renaming on `tests/CalDAVTest.php` (L57, 58)